### ISSUE TITLE
Send messages to group or channel from telegram-cli?

### ISSUE THREAD

**Issue by icepick3000** [vysheng/tg#1709]
Hi,
Does anyone know if it's possible to send messages from telegram-cli to a channel? 
My telegram-cli account is the admin of a channel.. but how do i post to it? 
I tried 
msg "channelname" "messagetext" 
msg "@channelname" "messagetext" 

but neither of them worked. 
Is this option not available?

**Comment by icepick3000** [comment]
Yes I tried it without.. doesn't help.. I am wondering if telegram-cli even supports groups/channels.

**Comment by Paulchen232** [comment]
I have try it too and the message arrive.

Have you a space in the channels name?

**Comment by icepick3000** [comment]
Nope, no spaces in the channelname.. 
Let me know if it works for you..

**Comment by Paulchen232** [comment]
I don't need it but it works.Have you use the argument -W? Arrives a error message?
